### PR TITLE
Fixed missing cache keys when loading profile page on first install

### LINKED ISSUE #944: Profile view does not load on first load
**Describe the bug**

**To Reproduce**
1. Reinstall the app
2. Click on profile view
3. See that the information do not load

**Expected behavior**
Informations should load when opening the page

**Screenshots**
image

### PR DESCRIPTION
**Pull request by DylanApardian**
### ⁉️ Related Issue
Closes #944 

## 📖 Description
When launching the app right after it's installed, some cache keys don't exist.
When calling:
- `await _userRepository.getInfo(fromCacheOnly: true)`
- `await _userRepository.getPrograms(fromCacheOnly: true)`

we're getting the info and the programs exclusively from cache, even though they don't exist.

### 🧪 How Has This Been Tested?
- Running the unit tests
- Uninstalling the app, launching it with the new changes and observing that the profile page loads as expected. 

### ☑️ Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] If needed, I added analytics.
- [x] Make sure to add either one of the following labels: `version: Major`,`version: Minor` or `version: Patch`. 
    - *Not sure if this is `patch` or `minor`*
- [ ] Make sure golden files changes were reviewed and approved.

### CODE REVIEW

File: lib/core/managers/user_repository.dart [hunk 1]
```diff
@@ -257,8 +266,11 @@ class UserRepository {
   /// The information from the [CacheManager] is loaded than updated with the results
   /// from the [SignetsApi].
   Future<ProfileStudent> getInfo({bool fromCacheOnly = false}) async {
```
Comment (XavierPaquet-Rapold) [inline lib/core/managers/user_repository.dart:268]:
```suggestion
  Future<ProfileStudent?> getInfo({bool fromCacheOnly = false}) async {
```

Comment (XavierPaquet-Rapold) [reply lib/core/managers/user_repository.dart:268]: Et à la ligne 283:
```
if (fromCacheOnly) {
      return _info;
}
```

**Review by XavierPaquet-Rapold — CHANGES_REQUESTED** [review]
J'ai créé une solution qui permet de changer uniquement 2 lignes

File: lib/core/managers/user_repository.dart (hunk 1 above)
Comment (DylanApardian) [reply lib/core/managers/user_repository.dart]: done

**Review by XavierPaquet-Rapold — APPROVED** [review]

**Review by MysticFragilist — APPROVED** [review]
LGTM! 💯

**Pull request merged**